pkg/monitoring/vms/prometheus/prometheus.go

```diff
@@ -363,6 +375,26 @@ func updateNetwork(vmi *k6tv1.VirtualMachineInstance, vmStats *stats.DomainStats
 	}
 }
 
+func updateTelemetry(nodeName string, vmis []*k6tv1.VirtualMachineInstance, ch chan<- prometheus.Metric) {
```

Comment:
I don't think that this is `telemetry` specific. In general these things are or will be valuable for monitoring. No change required.

Reply:
Not sure I fully understand your point, but I surely agree this is not Telemetry specific, so I'll rename this function, or squash in the calling site.